src/libraries/pkg/Directory.Build.props

```diff
@@ -15,7 +15,7 @@
 
   <!-- Packages opt-in to automatic RID-specific builds by placing a *.RID.props next to their project
        that defines the OfficialBuildRID item: all RIDs targeted by the package -->
-  <Import Project="$(MSBuildProjectDirectory)\*.rids.props" />
+  <Import Project="$(MSBuildProjectDirectory)\*.rids.props" Condition="'$(BuildingAnOfficialBuildLeg)' != 'true' or '$(RuntimeFlavor)' != 'Mono'"/>
```

Comment:
Why did we have to add this?

Reply:
I noticed that the new collapsed legs were producing the native rid specific packages, like: 'runtime.native.RID.native.System.IO.Ports` and we were publishing that into the intermediate unsigned assets (coming from the mono legs) which the intention of those is to just produce runtime packs. So at the end we would be signing two packages with the same name (because they were put in a different folder in the intermediate unsigned assets) and publishing whichever of the two wins at the end or causing failures like duplicate package entry, etc. So I think these should just be produced when building locally or using RuntimeFlavor coreclr.